firmware/integration/rusefi_config.txt

```diff
@@ -407,37 +408,37 @@ injector_s injectorSecondary
 
 bit isForcedInduction;Does the vehicle have a turbo or supercharger?
 bit useFordRedundantTps;On some Ford and Toyota vehicles one of the throttle sensors is not linear on the full range, i.e. in the specific range of the positions we effectively have only one sensor.
-bit lambdaProtectionEnable
+bit lambdaProtectionEnable;This enables the beta lambda protection. At present this feature is still in development.
```

Comment:
don't bother mentioning beta, it works fine

Reply:
(we'll also never remember to remove that note)

Reply:
If we are sure it works fine then no issues, will remove the beta.